airflow/providers/google/cloud/transfers/gcs_to_bigquery.py

```diff
@@ -300,6 +309,8 @@ def execute(self, context):
                 encoding=self.encoding,
                 src_fmt_configs=self.src_fmt_configs,
                 encryption_configuration=self.encryption_configuration,
+                labels=self.labels,
```

Comment:
Can you add a test in https://github.com/apache/airflow/blob/master/tests/providers/google/cloud/transfers/test_gcs_to_bigquery.py please, otherwise LGTM

Reply:
@kaxil I've added tests, let me know if they are acceptable.